frontend/src/static/js/v_authorship.js

```diff
@@ -388,6 +408,10 @@ window.vAuthorship = {
     this.removeAuthorshipHashes();
   },
 
+  updated() {
+    this.$store.commit('updateIsLoadingOverlayEnabled', false);
```

Comment:
If we decided to make the attribute global, we may not able to make use of the `updated` life hook right? This may accidentally terminate a loading overlay of another component?

Reply:
Changed loading overlay implementation